### DISCUSSION TITLE
Server list not displayed in Front-end. (Question & Self-Answer)

### DISCUSSION THREAD

**Discussion by Pascal62223** [equinor/witsml-explorer#99] [Q&A] [answered]
Hi,
    I've followed the "Contributing.md" guide to get WE installed / build and running on my machine.
    Everything is going good, and if I connect to my local MongoDB I can see that my default WITSML server is properly registered.  However when starting the front-end, the list of server remains empty. 

     Executing the "getAllServers()" fact in the mongodb test repository do returns the list of server (s) that have been added previously.
     
     The issue was the name of the collection set in the secrets.json and in the 'mysettings.json' : Both need to be the same :-) . I assume such configuration would benefit to be shared or discoverable in the future to keep test always in sync with source definition.

    Just a comment for new on-comers in case they face same issue.

**Comment by marmid74** [comment, marked as answer]
Updated inconsistence in documentation about [MongoDb in Contribution.md](https://github.com/equinor/witsml-explorer/blob/main/CONTRIBUTING.md#using-mongodb-for-storing-urls-to-witsmlservers), so hopefully new on-comers avoid this issue.
